### PR TITLE
FPET-706_C47A

### PR DESCRIPTION
**Pull request by remya-james**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java [hunk 1]
```diff
@@ -235,6 +235,12 @@ private static void filterDraftOrderForNewCases(String eventId, List<Element<Dra
     }
 
     public Map<String, Object> removeDraftOrderAndAddToFinalOrder(String authorisation, CaseData caseData, String eventId) {
+        try {
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java:238]: Please remove this logger

File: src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java [hunk 2]
```diff
@@ -271,6 +277,7 @@ public Map<String, Object> removeDraftOrderAndAddToFinalOrder(String authorisati
                     draftOrder = getUpdatedDraftOrder(draftOrder, caseData, loggedInUserType, eventId);
                 } else {
                     draftOrder = getDraftOrderWithUpdatedStatus(caseData, eventId, loggedInUserType, draftOrder);
+                    log.info("******after setting getDraftOrderWithUpdatedStatus" + draftOrder.getJusticeLegalAdviserFullName());
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java:280]: This one as well pls remove

File: src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java [hunk 3]
```diff
@@ -377,6 +384,13 @@ private Element<OrderDetails> convertDraftOrderToFinal(String auth, CaseData cas
                 .build();
         } else {
             manageOrderService.populateChildrenListForDocmosis(caseData);
+            caseData = manageOrderService.updateOrderFieldsForDocmosis(draftOrder,caseData);
+            try {
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java:388]: pls remove this logger, case data should not be printed.

File: src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java [hunk 4]
```diff
@@ -1684,7 +1684,9 @@ public Map<String, Object> getCaseData(String authorisation, CaseData caseData,
             if (CreateSelectOrderOptionsEnum.standardDirectionsOrder.equals(selectOrderOption)) {
                 caseData = populateJudgeName(authorisation, caseData);
             }
+
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java:1687]: Remove this orders

File: src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java [hunk 5]
```diff
@@ -2329,4 +2331,22 @@ && isNotEmpty(caseData.getManageOrders().getServeOrderAdditionalDocuments())) {
...
+        if (C100_CASE_TYPE.equalsIgnoreCase(CaseUtils.getCaseTypeOfApplication(caseData))) {
+            caseData = caseData.toBuilder()
+                    .judgeOrMagistratesLastName(draftOrder.getJudgeOrMagistratesLastName())
+                    .justiceLegalAdviserFullName(draftOrder.getJusticeLegalAdviserFullName())
+                    .magistrateLastName(draftOrder.getMagistrateLastName())
+                    .dateOrderMade(draftOrder.getDateOrderMade() != null ? draftOrder.getDateOrderMade() : draftOrder.getDateOrderMade())
+            .build();
+
+        }
+        log.info("******justiceLegalAdviserFullName" + caseData.getJusticeLegalAdviserFullName());
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java:2345]: Pls remove the logger

File: build.gradle [hunk 6]
```diff
@@ -587,6 +587,9 @@ def sonarExclusions = [
   '**uk/gov/hmcts/reform/prl/utils/*',
   '**uk/gov/hmcts/reform/prl/Application.*',
   '**/uk/gov/hmcts/reform/prl/services/TestingSupportService.java',
+  '**/uk/gov/hmcts/reform/prl/services/ManageOrderService.java',
```
Comment (priteshkr) [inline build.gradle:590]: Why you have excluded these files ?

**Review by yogendraU — APPROVED** [review]

**Pull request merged**